cea/plots/demand/energy_balance.py

```diff
@@ -9,6 +9,14 @@
 import pandas as pd
 import numpy as np
 
+__author__ = "Gabriel Happle"
+__copyright__ = "Copyright 2018, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Gabriel Happle","Jimeno A. Fonseca"]
```

Comment:
needs a comment after the space

Reply:
`["Gabriel Happle","Jimeno A. Fonseca"]` - there should be a space after the comma. pep8. (PyCharm hints at this with squiggly lines, similar to Word. ideally, the less such complaints that are unnecessary, the more information they provide, so it's good to fix the easy stuff by habit)